pkg/cidata/cidata.TEMPLATE.d/boot/25-guestagent-base.sh

```diff
@@ -41,9 +47,13 @@ name="lima-guestagent"
 description="Forward ports to the lima-hostagent"
 
 command=${LIMA_CIDATA_GUEST_INSTALL_PREFIX}/bin/lima-guestagent
-command_args="daemon --debug=${LIMA_CIDATA_DEBUG} --vsock-port \"${LIMA_CIDATA_VSOCK_PORT}\" --virtio-port \"${LIMA_CIDATA_VIRTIO_PORT}\""
-command_background=true
-pidfile="/run/lima-guestagent.pid"
+command_args="daemon --debug=${LIMA_CIDATA_DEBUG} \
+--docker-sockets \"${LIMA_CIDATA_DOCKER_PORT_MONITOR_SOCKETS}" \
+--containerd-sockets \"{LIMA_CIDATA_CONTAINERD_PORT_MONITOR_SOCKETS}" \
+--kubernetes-configs \"{LIMA_CIDATA_KUBERNETES_SERVICE_WATCHER_CONFIGS}" \
+--vsock-port \"${LIMA_CIDATA_VSOCK_PORT}\" \
+--virtio-port \"${LIMA_CIDATA_VIRTIO_PORT}\""
+command_background=true pidfile="/run/lima-guestagent.pid"
```

Comment:
Keep them on separate lines:

```suggestion
command_background=true
pidfile="/run/lima-guestagent.pid"
```